ios/src/TGMapViewController.h

```diff
@@ -46,31 +47,50 @@ struct TileID;
 @interface TGMapViewController : GLKViewController <UIGestureRecognizerDelegate>
 
 @property (assign, nonatomic) BOOL continuous;
-@property (weak, nonatomic, nullable) id<TangramGestureRecognizerDelegate> gestureDelegate;
+@property (weak, nonatomic, nullable) id<TGRecognizerDelegate> gestureDelegate;
+@property (weak, nonatomic, nullable) id<TGMapViewDelegate> mapViewDelegate;
 
 // The following are computed property. They return sensible defaults when the above .map property is nil
-@property (assign, nonatomic) TangramCameraType cameraType;
-@property (assign, nonatomic) TangramGeoPoint position;
+@property (assign, nonatomic) TGCameraType cameraType;
+@property (assign, nonatomic) TGGeoPoint position;
 @property (assign, nonatomic) float zoom;
 @property (assign, nonatomic) float rotation;
 @property (assign, nonatomic) float tilt;
 
 - (void)renderOnce;
 
-- (void)animateToPosition:(TangramGeoPoint)position withDuration:(float)seconds;
+NS_ASSUME_NONNULL_BEGIN
+
+- (void)loadSceneFile:(NSString*)path;
+
+- (void)loadSceneFileAsync:(NSString*)path;
+
+- (void)queueSceneUpdate:(NSString*)componentPath withValue:(NSString*)value;
+
+NS_ASSUME_NONNULL_END
+
+- (void)pickFeaturesAt:(CGPoint)screenPosition;
+
+- (void)requestRender;
+
+- (CGPoint)lngLatToScreenPosition:(TGGeoPoint)lngLat;
```

Comment:
Those are c-type structs, assuming them non-null with the macro doesn't seem to make too much sense since those value would get copied.